esp/bindings/SOAP/Platform/soapbind.hpp

```diff
@@ -148,6 +151,12 @@ class esp_http_decl CSoapRequestBinding : public CSoapComplexType,
     void setThunkHandle(void * val){thunk_=val;}
     void * getThunkHandle(){return thunk_;}
 
+    void setConnectTimeOutMs(unsigned timeout){connectTimeoutMs = timeout;}
```

Comment:
OK. There is no base implementation at this point.

There is the pure virtual in the interface but this is the first level it is being implemented.

My point was that I can see using override for the  implementation of a pure virtual interface functions, but since they are used constantly all over the HPCC code base, I'd like to get consensus on it becoming an HPCC wide standard.